### PR TITLE
feat : Added badge component

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added Badge Components  to component library

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
Looks nice. Good to merge !

File: Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    align-items: center;
+    justify-content: center;
+    margin: 10px;
+}  
+  
+.badge-icon {
+    width: 50px;
+    height: 50px;
+    color: #333;
+    background-color: #ddd;
```
Comment (sks-21) [inline Components/badge.css:13]: Just one suggestion, use generic color variables

Comment (Omkar-Chaskar) [reply Components/badge.css:13]: Thanks , in next PR i will do use variables.

**Review by munjalSaksham71 — APPROVED** [review]
Looks Cool. You can merge it

**Review by ultronjs — APPROVED** [review]
The code looks clean.Same suggestion as Shashank,hopefully you will fix that in future PR's

**Pull request merged**